mountpoint-s3/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### Other changes
 
 * Mountpoint now retries S3 requests up to a total of 10 attempts (up from 4), which should make file operations more robust to transient failures or throttling. The maximum number of attempts can be overridden by setting the `AWS_MAX_ATTEMPTS` environment variable. ([#830](https://github.com/awslabs/mountpoint-s3/pull/830))
+* Mountpoint for Amazon S3 now supports specifying an AWS Key Management Service (AWS KMS) key for server-side encryption with KMS (SSE-KMS) when mounting an S3 bucket or prefix. ([#534](https://github.com/awslabs/mountpoint-s3/issues/534), [#807](https://github.com/awslabs/mountpoint-s3/issues/807))
```

Comment:
We link to the pull requests that introduced the change here.

Reply:
why? :) issue contains links to the PR and its description is much more informative than any of the PRs

Reply:
put the PR link anyways!